src/transformers/feature_extraction_sequence_utils.py

```diff
@@ -240,6 +223,8 @@ def pad(
             for key, value in outputs.items():
                 if key not in batch_outputs:
                     batch_outputs[key] = []
+                if value.dtype is np.dtype(np.float64):
+                    value = value.astype(np.float32)
```

Comment:
Instead of adding dtype customization to `.pad()`, just always use `np.float32`, until we really need `float64` support for some new models.